### ISSUE TITLE
Number of false alarms and phrasing of warning message

### ISSUE THREAD

**Issue by stefanschmidt** [objective-see/RansomWhere#4]
I really like the idea of generic ransomware detection and I'm aware that's it's non-trivial to achieve a good balance between over-blocking and under-blocking.

There are two situations where I think that there might be room for improvement though.

1. After every Chrome update I usually get at least one false alarm:

<img width="532" alt="ransomwhere-google-chrome-false-alarm" src="https://github.com/objective-see/RansomWhere/assets/303215/3052c528-768b-4a44-b1cd-2cb8b57962c5">

> Any process's binary signed by Apple proper is inherently trusted. (...) Note, this does not include binaries signed with an Apple developer ID! ([source](https://objective-see.org/blog/blog_0x0F.html))

The binary is signed but of course not by Apple proper:

<img width="804" alt="chrome-signature" src="https://github.com/objective-see/RansomWhere/assets/303215/085f18f0-d6bc-4c68-9731-ec3f326bc80f">

I'm wondering, could it be meaningful to have a feature to whitelist specific Apple developer IDs instead of a binary and thereby avoiding the reoccuring false alarm? I would of course be responsibility of the user to decide which developers they trust.

2. When installing or upgrading large Homebrew packages from source (e.g. Rust or GCC) there are usually at least have a doven false alarms during extraction and compilation:

<img width="532" alt="bsdtar-false-alarm" src="https://github.com/objective-see/RansomWhere/assets/303215/c79db12c-1bae-4c4c-a06a-d1afb7c1ce26">

This occurs even though `bsdtar` is signed by Apple proper:

<img width="804" alt="bsdtar-signature" src="https://github.com/objective-see/RansomWhere/assets/303215/51f7badd-7169-4756-b565-bbfda3721f6b">

**Comment by andrewcrook** [comment]
Yes I am having this with other chrome based browsers as well as a few other apps.

Obviously with a browser you want to allow certain events but not others.

Would it be useful to have a way to exclude folders and files (file level probably best recommended).
Maybe rules to combine with allowing certain apps and devs by using code-signings?

**Comment by andrewcrook** [comment]
Great thanks
